docs/readme.md

```diff
@@ -28,6 +28,22 @@ launchChromeAndRunLighthouse('https://example.com', flags).then(results => {
 });
 ```
 
+## Performance-only Lighthouse run
+
+Many modules consuming Lighthouse are only interested in the performance numbers. 
+Lighthouse ships with a [performance-only config](https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/config/perf.json) that you can use:
+
+```js
+const perfConfig: any = require('lighthouse/lighthouse-core/config/perf.json');
+
+// ...
+
+lighthouse(url, flags, perfConfig)
+```
+
+You can also craft your own config (e.g. [plots.json](https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/config/plots.json)) for completely custom runs.
```

Comment:
https://github.com/GoogleChrome/lighthouse/tree/master/docs/recipes/custom-audit#the-audit-gatherer-and-config might be a better link.